index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
+  </button>
+</div>
+      </div>
+      </section>
+  </section>
+
+
+  <section class="background-image">
+    <section class="image-content">
+        <div class="image-cont-holder">
+        <h1 class="image-h1">
+            Profesional Art Printing Data
+        </h1>
+        <p class="para">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+
+        <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+    See Project
+</button>
+</div>
+    </div>
+    </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+
+</section>
+</section>
+<section class="about-me">
+  <section class="about-me-page">
+    <div class="about-me-all">
+      <h1 class="about-me-h1">
+        About me
+      </h1>
+      <div class="about-me-para-div">
```

Comment:
- [ ] Kindly change the h1 to a lower hx tag because following semantic HTML on one page it should be only one h1 and it should be in the beginning. Keep rocking💪